client/account-recovery/reset-password/reset-password-confirm-form/index.jsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+import { identity } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import FormTextInput from 'components/forms/form-text-input';
+import FormLabel from 'components/forms/form-label';
+import FormButton from 'components/forms/form-button';
+import { STRONG_PASSWORD } from 'lib/url/support';
+
+const ResetPasswordConfirmForm = ( props ) => {
+	const {
+		translate,
+		showPassword,
+	} = props;
+
+	return (
+		<Card>
+			<h2 className="reset-password-confirm-form__title">{ translate( 'Reset your password' ) }</h2>
+			<FormLabel className="reset-password-confirm-form__text-input-label" htmlFor="password">
+				{ translate( 'New password' ) }
+			</FormLabel>
+			<a className={ classNames( 'reset-password-confirm-form__show-hide-toggle', { show: showPassword } ) } href="#">
```

Comment:
Nice tip! It eliminated a lot of redundant code. Applied in 17712b0c75f2914daa31b674ad67cf4a2980d211.